bbc/simorgh
# Remove image endpoints and default originCode

Resolves #793 

_This PR removes the base path from the image URLs in the fixture data, this aligns with the Optimo data as it only supplies locators. This PR also refactors the originCode [logic](https://github.com/BBC-News/simorgh/pull/798/files#diff-445cfd09f5ef4bfbe3544be7b7f2da00R25) so that null or empty string origin codes default to `cpsdevpb` this is needed because Optimo currently does not supply an originCode and the work is not yet scoped. As part of the originCode refactor I ensured the requirement to return the locator value for the originCode of `pips` was still met._


## Test instructions ##
Unit test and Cypress tests should cover these changes, I also suggest a basic manual regression to ensure images load
